docs/content/en/docs/operator-manual/piped/configuring-event-watcher.md

```diff
@@ -11,24 +11,38 @@ To enable [EventWatcher](/docs/user-guide/event-watcher/), you have to configure
 ### [required] Grant write permission
 The [SSH key used by Piped](/docs/operator-manual/piped/configuration-reference/#git) must be a key with write-access because piped needs to commit and push to your git repository when any incoming event matches.
 
-### [optional] Settings for watcher
-The Piped's behavior can be finely controlled by setting the `eventWatcher` field.
+### [required] Specify Git repositories to be observed
```

Comment:
I think by default, all tasks are considered as required so we can remove the `[required]` tag.
The optional tasks should include the `[optional]` tag explicitly.

Reply:
Makes sense. Let me do that